specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2513,59 +2509,6 @@
         }
       }
     },
-    "ManagedServiceIdentity": {
```

Comment:
Why are we removing ManagedServiceIdentity here? Will it come in another api-version?

Reply:
Yes, managed identity is not ready for GA yet, we will include this in next preview version.